packages/atomic/create-atomic/scripts/packageTemplate.json

```diff
@@ -3,6 +3,11 @@
   "description": "{{titleCase project}} project",
   "version": "0.1.0",
   "private": true,
+  "main": "dist/index.cjs.js",
```

Comment:
`cjs.js` wat?
do you mean `.cjs`?

Reply:
No! That's what Stencil outputs and recommends when you don't have the main set
<img width="938" alt="Screen Shot 2022-04-13 at 9 57 24 AM" src="https://user-images.githubusercontent.com/4923043/163197048-104c31da-5b30-4993-b11f-64c15f77726a.png">
<img width="190" alt="Screen Shot 2022-04-13 at 9 56 54 AM" src="https://user-images.githubusercontent.com/4923043/163197051-adc49af0-3c13-4453-b89e-6d3088e5346b.png">

Reply:
oh ok that's funny.